templates/docs/patterns/search-and-filter.md

```diff
@@ -50,21 +50,21 @@ When a click event is triggered on the selected-count element you need need to t
   <p class="p-notification__response" role="status">You will have to handle the overflowing chip count yourself. Unfortunately that is out of scope for this example but this can be handled for you by the <a href="#react">React component</a>.</p>
 </div>
 
-### Search prompt
+## Search prompt
 
 When you type in the search input the behaviour should be that the available selections are filtered to match the input. Also, a raw string input should be made available.
 
 <div class="embedded-example"><a href="/docs/examples/patterns/search-and-filter/with-search-prompt/" class="js-example">
 View example of the search and filter with search prompt pattern
 </a></div>
 
-#### Functionality
+### Functionality
 
 <div class="p-notification--information">
   <p class="p-notification__response" role="status">You will have to handle filtering the available chips yourself by removing or hiding chips that do not match. Unfortunately that is out of scope for this example but this can be handled for you by the <a href="#react">React component</a>.</p>
 </div>
 
-## Import
+# Import
```

Comment:
I guess something was messed up here in hierarchy. Would be good to review this page and check if others are fine.

```suggestion
## Import
```